### PR TITLE
TFKUBE-247: Prepare the documentation for GA release

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-site_name: Terraform for Atlassian DC on Kubernetes
+site_name: Deployment Automation for Atlassian DC on K8s
```
Comment (nanux) [inline docs/mkdocs.yml:1]:
```suggestion
site_name: Deployment Automation for Atlassian DC on Kubernetes
```

I prever the longer version, at least here for SEO.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 # Change Log
 
-## v0.0.2-beta
+## 1.0.0
+
+**Release date:** 2022-01-31
+
+* Improved documentation
+* Changed the defualt values in configuration template
```
Comment (nanux) [inline CHANGELOG.md:8]:
```suggestion
* Changed the default values in the configuration template
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+* Resume the Bamboo server after installation
+* Improved install and uninstall scripts
+* Made the product version, Bamboo software and Bamboo agent resources configurable
```
Comment (nanux) [inline CHANGELOG.md:11]:
```suggestion
* Added optional configuration for Bamboo version and resources available to Bamboo DC and Bamboo agents
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+* Improved the end to end tests
+* Changed the default product version (Bamboo software and Bamboo agent Helm charts) to `1.0.0`
```
Comment (nanux) [inline CHANGELOG.md:13]: Can we list the product version that we have updated to as well?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,19 @@
...
+* Changed the maximum length for environment name to 24 characters
```
Comment (nanux) [inline CHANGELOG.md:14]: We also now restrict the environment to lowercase letters as well.

**Review by nanux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (nghazali) [reply CHANGELOG.md:13]: I removed this line as this change is done as part of version `0.0.2` and already we have it in change log

**Review by nanux — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (nghazali) [reply CHANGELOG.md]: done

**Review by errcode1202 — APPROVED** [review]

**Pull request merged**